les/handler.go

```diff
@@ -733,7 +733,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 				// Retrieve the requested block's receipts, skipping if unknown to us
 				var results types.Receipts
 				if number := rawdb.ReadHeaderNumber(pm.chainDb, hash); number != nil {
-					results = rawdb.ReadReceipts(pm.chainDb, hash, *number)
+					results = rawdb.ReadReceipts(pm.chainDb, hash, *number, pm.blockchain.Config())
```

Comment:
Do we need the full receipts here with all fields derived, or is the raw one with only consensus fields enough?